ui/pages/confirmations/selectors/preferences.ts

```diff
@@ -0,0 +1,8 @@
+export type RootState = {
+  metamask: {
+    useTransactionSimulations?: boolean;
+  };
+};
+
+export const getUseTransactionSimulations = (state: RootState) =>
```

Comment:
Traditionally I think all the preference selectors have gone in `selectors.js`, but I'm a huge fan of more modular selector files.

Should we go with `selectUseTransactionSimulations` for clarity as I think Redux recommend that but it makes it clear what is a selector and what is a util or helper.

Reply:
PR logic lgtm! I agree I'd be good to prefix with select
ref: https://redux.js.org/style-guide/#name-selector-functions-as-selectthing

Reply:
I made the update